app/initialize.js

```diff
@@ -70,20 +72,24 @@ function renderTable(data) {
   var tableHeader = repoTable.append('thead').append('tr');
   tableHeader.append('th').html('Block')
     .on('click', function() {
-      renderRows(data, tableBody, 'name');
+      ascending = sortedBy === 'name' ? !ascending : false;
```

Comment:
Is this repeated line worth putting in it's own function/do you have a better idea on how to toggle sorting order?